study.md

```diff
@@ -50,7 +51,8 @@ Does `proxima_centauri.print_mass` run? Why or why not? If so, then what is the
 output? If not, then how can it be fixed?
 
 ```md
-<!-- your answer here -->
+Nope. mass_kg isn't a variable that is available globally.
```

Comment:
It's not available because local to the constructor and undefined in the `print_mass` method.  Global variables start with `$`.